src/main/java/com/minecolonies/coremod/entity/ai/citizen/research/EntityAIWorkResearcher.java

```diff
@@ -23,9 +23,9 @@
     public static final int STUDY_DELAY = 60;
 
     /**
-     * XP gained per study position
+     * base XP gained per study position
      */
-    private static final double XP_PER_STUDYPOS = 0.7;
+    private static final double XP_PER_STUDYPOS = 3.125;
```

Comment:
I think XP and studies/day are still off, currently we have a study delay of just 60 ticks(3seconds) + walking time inbetween, which is likely more than just ~3 studies a day you need for getting to the average 10xp base.

A minecraft day is 10 minutes, ~3 of those are spent at getting to or from work, so lets say 7mins effective working time. If they now study for 3s + ~20s walking time you get a total of 18 study actions a day.
Following the baseline of 10Xp a day a worker should get, you'll end up giving them 10/18 = ~0.55 xp per study action.

I think we should do two things here: 
First increase the study delay, 60 ticks is very fast I think they should be looking at a "book" near a bookcase for about 60 seconds instead which would be 60 * 20 ticks.

When we study every 60 seconds and estimate another 20seconds of walking time we get 80 seconds per study, with the estimated minecraft work day of 7minutes that is ~ 5 Study actions per day. 

At 10 Xp Base and 5 study actions a day you'd give it 2 XP per study action.

Reply:
Will do!  I misread the XP calculation, I thought that

`localXp += localXp * (intelligenceLevel / 100.0);`

was actually 

`localXp = localXp * (intelligenceLevel / 100.0);`

(in which case I should have given them 31.25 xp per action).  
I'll do the things you suggested!